pkg/cluster/runcommand.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-10-01/compute"
+	"github.com/Azure/go-autorest/autorest/to"
+
+	"github.com/openshift/openshift-azure/pkg/api"
+)
+
+func (u *simpleUpgrader) RunCommand(ctx context.Context, oc *api.OpenShiftManagedCluster, scaleSetName, instanceId string, parameters compute.RunCommandInput) (compute.RunCommandResult, *api.PluginError) {
+	return compute.RunCommandResult{}, nil
+}
+
+func (u *simpleUpgrader) RestartDocker(ctx context.Context, oc *api.OpenShiftManagedCluster, scaleSetName, instanceId string) (compute.RunCommandResult, *api.PluginError) {
+	params := compute.RunCommandInput{
+		CommandID: to.StringPtr("RunShellScript"),
+		Script:    to.StringSlicePtr([]string{"systemctl restart docker.service"}),
```

Comment:
just nit: all these are systemd services. we could have a wrapper for systemd managed service. We could trigger a restart, status, reload, etc.